central/integrationhealth/datastore/singleton.go

```diff
@@ -13,7 +18,13 @@ var (
 )
 
 func initialize() {
-	ad = New(rocksdb.New(globaldb.GetRocksDB()))
+	var storage store.Store
+	if features.PostgresDatastore.Enabled() {
+		storage = postgres.New(context.TODO(), globaldb.GetPostgres())
```

Comment:
Is there any reason to use `context.TODO()` rather than `context.Background()` or passing a context to `initialize()` here ?
(note: a quick look into the `initialize`/`Singleton` call stack suggests `context.Background()` might be the less painful alternative)

Reply:
Em, we are not ready to a real context yet. This TODO is in align with other postgres stores so that they can be replace together. https://issues.redhat.com/browse/ROX-9663